src/services/patchwork.rs

```diff
@@ -50,59 +50,63 @@ pub fn start<
                         map_index: 0,
                         conn_id: None,
                     });
-                match &patchwork.maps[anchor.map_index].peer_connection {
-                    Some(_) => match msg.packet {
-                        Packet::Unknown => {}
-                        _ => {
-                            trace!(
-                                "Routing packet from conn_id {:?} through anchor",
-                                msg.conn_id
-                            );
-                            player_state.anchored_move_and_look(
-                                msg.conn_id,
-                                extract_player_position((&msg.packet).clone()),
-                                None,
-                            );
-                            messenger.send_packet(anchor.conn_id.unwrap(), msg.packet.clone());
-                        }
-                    },
-                    None => {
-                        trace!("Routing packet from conn_id {:?} locally", msg.conn_id);
-                        gameplay_router::route_packet(
-                            msg.packet.clone(),
-                            msg.conn_id,
-                            player_state.clone(),
-                            sender.clone(),
-                        );
-                    }
```

Comment:
Why delete this? Won't all non-positional packets get ignored now?

Reply:
Ah whoops, I was just trying to avoid handling position packets that are out of bounds so I moved it to after we check the map position but you're right that this will ignore non position packets...